js/index.js

```diff
@@ -0,0 +1,180 @@
+const fibTextEl = document.getElementById("fib-text-el");
+const inputEl = document.getElementById("input-el");
+const calcBtn = document.getElementById("calc-btn");
+const input = document.querySelector("input");
+const spinner1El = document.querySelector("#spinner1-el");
+const spinner2El = document.querySelector("#spinner2-el");
+const renderResult = document.getElementById("render-result");
+const over50 = document.getElementById("over50");
+const checkbox = document.getElementById("flexCheckDefault");
```

Comment:
Why some querySelector and other getElementById? Should keep it consistent. I recommend you stick with querySelector